### ISSUE TITLE
Some packages from BioC not available within Packrat projects

### ISSUE THREAD

**Issue by christianholland** [rstudio/packrat#478]
Hi,
I started to work with packrat projects. However, when I want to install BioConductor packages I receive for some packages error messages that the package could not be installed.

```
# Install package with biocLite
BiocInstaller::biocLite("gskb")

# output
BioC_mirror: https://bioconductor.org
Using Bioconductor 3.7 (BiocInstaller 1.30.0), R 3.5.0 (2018-04-23).
Installing package(s) ‘gskb’

   package ‘gskb’ is available as a source package but not as a binary
```

When I want to install those packages outside a packrat project this error doesn't appear.

**Comment by kevinushey** [comment]
I see this if I attempt to install the package outside of Packrat:

```r
> biocLite("gskb", type = "binary")
BioC_mirror: https://bioconductor.org
Using Bioconductor 3.7 (BiocInstaller 1.30.0), R 3.5.0 (2018-04-23).
Installing package(s) 'gskb'

   package 'gskb' is available as a source package but not as a binary

Warning: package 'gskb' is not available (as a binary package for R version 3.5.0)
```

So this appears to be something going on with BioConductor. You can explicitly request the source version with:

```r
biocLite("gskb", type = "source")
```

**Comment by christianholland** [comment]
Many thanks for pointing to the `type` argument. This enables the installation
